src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -611,6 +611,22 @@ namespace UtilityRoutines {
         return calctime;
     }
 
+    bool isKeyRegexLike(std::string_view key)
+    {
+        return key.find_first_of("*+?()|[]\\") != std::string_view::npos;
+
+        // DataOutputs does it like this
+        // bool is_simple_string = true;
+        // for (auto const &c : key) {
+        //     if (c == ' ' || c == '_' || (std::isalnum(c) != 0)) {
+        //         continue;
+        //     }
+        //     is_simple_string = false;
+        //     break;
+        // }
+        // return !is_simple_string;
+    }
```

Comment:
So I started with the find_first_of, then I (much) later found that DataOutputs already does something similar. Not sure which is best, but both should end up using the same. thoughts?

https://github.com/NREL/EnergyPlus/blob/6e02a12bb76472532821e6eab4f22eee748afbc6/src/EnergyPlus/DataOutputs.cc#L138-L157

Reply:
cf ec075e14eaec9d2000ac38641bd6e19ec9df8f92...ee41ce7ea23a866c3cfb7af5286ebb669708a59e